packages/dds/shared-object-base/src/test/createSharedObjectKind.spec.ts

```diff
@@ -60,8 +60,7 @@ class SharedFooFactory implements IChannelFactory<IFoo> {
 			foo: "bar",
 			attributes: this.attributes,
 			id,
-			minVersionForCollab: (runtime as IFluidDataStoreRuntimeInternalConfig)
```

Comment:
Now that `minVersionForCollab` is not optional in `IFluidDataStoreRuntimeInternalConfig` but still doesn't exist in `IFluidDataStoreRuntime`, casting `runtime` doesn't work without `as unknown as IFluidDataStoreRuntimeInternalConfig` because now the compiler complains about the fact that `minVersionForCollab` wouldn't be available. Since the implementation `FluidDataStoreRuntime` does have the property as required, the new cast felt more appropriate and that's why I removed `minVersionForCollab` from `IFluidDataStoreRuntimeInternalConfig`.